webpack.config.js

```diff
@@ -48,7 +49,7 @@ const config = {
               sourceMap: true,
               postcssOptions: {
                 plugins: [
-                  "postcss-preset-env"
+                  postcssPresetEnv({ enableClientSidePolyfills: false })
```

Comment:
Added this because of the `:focus-visible` polyfill getting in the way with its `.focus-visible` class: https://github.com/csstools/postcss-plugins/tree/main/plugin-packs/postcss-preset-env#plugins-that-need-client-library